android/libraries/rib-coroutines-test/src/main/kotlin/com/uber/rib/core/TestRibCoroutineScopes.kt

```diff
@@ -13,6 +13,8 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+@file:Suppress("invisible_reference", "invisible_member")
```

Comment:
This is gross. Let's capture the reason by documenting youtrack issue. Also, did you see anything about a longer term direction on how to think about this? Are they not really supporting friendpaths anymore?

Reply:
Agreed this is horrible. After much digging I think friend paths was never properly publicly supported/documented. I found it really hard to find any docs on this, even for the barebones `kotlinc` compiler. I basically dig into a lot of YouTrack issues to see the state of art on this and most issues are multiple years old. Not to mention Kotlin Gradle plugin. @jbarr21 might have some context on this as the original implementor of the friends Gradle config we use.

The alternatives I see are:
1. Don't access internal from other modules. But making stuff like `LazyCoroutineScope` public is a no-no.
2. (Untested, might be missing something) rethink module structure; create a third module for containing common components as `public` but don't expose it: import it as `implementation` in dependent modules. I'm just unsure if it's possible to do without generating public artifacts for this "third module", which we don't want. I think no.

So basically I don't like either strategy.

Reply:
Maybe there's some nice syntax for configuring friend paths in Gradle / Kotlin Gradle plugin as we do on BUCK and I'm just missing it :)